maui-toc.html

```diff
@@ -127,6 +127,7 @@
             <li><a href="/maui/Accordion/migration">Migrate from Xamarin.Forms</a></li>
             <li><a href="/maui/Accordion/appearance">Appearance</a></li>
             <li><a href="/maui/Accordion/bindablelayout">BindableLayout</a></li>
+            <li><a href="/maui/Accordion/accordion-events"></a></li>
```

Comment:
Add title and rename file as events

Reply:
Added the title and renamed the file as events.